### PR TITLE
Feature cbmc bump

### PR DESCRIPTION
**Pull request by hannes-steffenhagen-diffblue**
Bump CBMC and fix issues arising from it

### CODE REVIEW

File: gnat2goto/driver/driver.adb [hunk 1]
```diff
@@ -164,6 +191,75 @@ package body Driver is
          Append_Op (Start_Body, Initialization_Statement);
       end Initialize_CProver_Rounding_Mode;
 
+      procedure Initialize_CProver_Dead_Object;
```
Comment (xbauch) [inline gnat2goto/driver/driver.adb]: So these three procedure only differ in the `Identifier`?

Comment (hannes-steffenhagen-diffblue) [reply gnat2goto/driver/driver.adb]: pretty much. I can abstract if if you prefer?

Comment (xbauch) [reply gnat2goto/driver/driver.adb]: I'm not insisting.

**Comment by chrisr-diffblue** [comment]
I've confirmed with @martin-cs that it's ok to bump the submodule - so just needs CI sorting and second review.

**Comment by chrisr-diffblue** [comment]
BTW - If there is a specific reason for the submodule bump, (e.g. to pickup a fix/feature we need) it would be fantastic if that reason was included in the commit message for the commit that actually moves the submodule pointer :-)

**Pull request merged**